distro/run/qa/src/test/java/org/camunda/bpm/run/qa/util/SpringBootManagedContainer.java

```diff
@@ -267,27 +270,27 @@ protected static Integer windowsProcessId(Process process) {
     return null;
   }
 
-  private void createTestYml() {
+  public void createConfigurationYml(String filePath, File source) {
     try {
-      File baseYml = new File(SpringBootManagedContainer.class.getClassLoader().getResource(BASE_TEST_APPLICATION_YML).getFile());
-      File testYml = new File(new File(baseDirectory), APPLICATION_YML_PATH);
-      Files.copy(baseYml, testYml);
+      File testYml = new File(new File(baseDirectory), filePath);
+      Files.copy(source, testYml);
```

Comment:
Let's use `java.nio.file.Files` here.
This expects for example an `InputStream` as source (can be received with `#getResourceAsStream` from the class loader) and a `Path` as target (can be build easily by using `java.nio.file.Paths#get`).